src/plugins/vis_type_table/public/table_vis_response_handler.ts

```diff
@@ -48,7 +48,7 @@ export interface TableGroup {
 }
 
 export interface TableContext {
-  table?: Table;
```

Comment:
I assume it will be in later PRs, but we'll still need proper error handling for when the `visData` contains no `table` data.

Reply:
I will create a sub issue for this.

Reply:
I change it back.